.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,37 @@
+---
+name: Bug report
+about: Tell about typedoc's bug
+---
+<!--
+    Thank you for contributing to Typedoc! 
+    
+    First, please use a clear and descriptive title for the issue to identify the bug.
+
+    Please be as specific as possible! Provide codes, sreenshots, repositories, explanations, issues, links ... anything that could describe your point.
+-->
+
+- [ ] I have checked [issues with bug label](https://github.com/TypeStrong/typedoc/labels/bug) and found no duplicates
+
+## Expected Behavior
+<!--
+    According to your expectations, how is Typedoc supposed to work?
+-->
+
+## Actual Behavior
+<!--
+    What does Typedoc fail to do?  
+-->
+
+## Steps to reproduce the bug
+<!--
+    Tell us how we could reproduce the bug. Possible ways are providing:
+    - Typedoc command (typedoc --out ...)
+    - Configuration (typedoc.js or your custom option file)
+    - Code that causes a bug in the documentation generated
+    - Project (Github repository, CodePen, ...) where the issue can be reproduced.
+-->
+
+## Specifications
```

Comment:
Would `Environment` be a better word for this heading?

Reply:
Yeah. Both are fine, actually

Reply:
```suggestion
## Environment
```